### DISCUSSION TITLE
Neey 4A EK-24S4EB

### DISCUSSION THREAD

**Discussion by ThibautBascoulergue** [syssi/esphome-jk-bms#488] [Q&A]
Hello, I have a Neey EK-24S4EB and I saw in this post "https://github.com/syssi/esphome-jk-bms/issues/444" that it can be connected. Having an esp8266, which bluetooth module should I buy for everything to work? HC-05 or HC-06
THANKS

**Comment by chipsi** [comment]
No, too much work and ESP8266 is weak. Buy ESP32 module. You need change BLE YAML from GW- to EK-.

**Comment by Mgzeroman12** [comment]
Hi can anybody  give me a link to the Android app for Neey smart active balancer EK-24SEB thankyou
